staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -54,10 +54,13 @@ const (
 	CloudProviderName      = "azure"
 	rateLimitQPSDefault    = 1.0
 	rateLimitBucketDefault = 5
-	backoffRetriesDefault  = 6
-	backoffExponentDefault = 1.5
-	backoffDurationDefault = 5 // in seconds
-	backoffJitterDefault   = 1.0
+	// VMSS API limits are 180 queries per 3 minutes & 900 queries per 30 minutes per subscription
```

Comment:
this PR would add the cloud provider rate limit, it could hit the throttling value of cloud provider easily, and 180 per 3 minutes & 900 per 30 minutes are for the whole subscription. It's hard to tell the correct rate limit value per vmss cluster.

Reply:
Yes you are right.

Still those default values should prevent one cluster to hit the VMSS API subscription limits.

Reply:
those values are really depending on cluster setup, let's remove those comments from our codes.